src/components/MonthlyGood/MonthlyGoodDropInPaymentWrapper.vue

```diff
@@ -14,16 +14,16 @@
 				<kv-button
 					value="submit"
 					id="dropin-submit"
-					class="button"
-					:disabled="!enableConfirmButton || submitting"
-					@click.native="submitDropInMonthlyGood"
+					class="tw-mb-2"
+					:state="`${!enableConfirmButton || submitting ? 'disabled' : ''}`"
```

Comment:
Since this is using v-bind I think you can drop the template string part and use the contents directly:
```
:state="!enableConfirmButton || submitting ? 'disabled' : ''"
```